index.html

```diff
@@ -71,27 +71,82 @@ <h1 class="headerText">Welcome
                     <h2>Our Projects</h2>
                     <ul class="project-list">
                         <li class="soundboard">
-                            <span class="text">A soundboard made in Java</span>
+                            <span class="text">Android soundboard app (Java)</span>
                             <iframe src="https://www.youtube.com/embed/75eax_8SYyI"
                             width="300" height="200"></iframe>
                         </li>
```

Comment:
personally, I'd prefer the Rattler app on top, as that is a much larger undertaking and achievement :D